system-tests/src/tests/mark-as-spam.spec.ts

```
@@ -0,0 +1,88 @@
+  context1 = await browser.newContext({ storageState: "src/states/student1@example.com.json" })
+  context2 = await browser.newContext({ storageState: "src/states/student2@example.com.json" })
+  context3 = await browser.newContext({ storageState: "src/states/teacher@example.com.json" })
+})
+
+test.afterEach(async () => {
+  await context1.close()
+  await context2.close()
+  await context3.close()
+})
+
+test("Mark answer as spam in peer review", async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)